GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -155,10 +152,19 @@ private void OnGuiCreateBundleSelect()
 
             if (GUILayout.Button("Build AssetBundle"))
             {
+                var assetBundleBuildPath = EditorUtility.SaveFilePanel("Save AssetBundle", "", "", "");
```

Comment:
What is the right extension for these files? Should the file extension be the last parameter?